_pages/gedcom551-errata.md

```diff
@@ -0,0 +1,30 @@
+---
+title: GEDCOM 5.5.1 Errata
+permalink: /gedcom551-errata/
+sidebar:
+  nav: "errata"
+---
+# GEDCOM 5.5.1 Errata
+
+The following is the current list of errata for [the GEDCOM Standard, Release 5.5.1, published 15 November 2019](https://gedcom.io/specifications/ged551.pdf).
+
+|Page|Line|Heading| Original Text | Corrected Text | Notes |
+|----|----|-------|---------------|----------------|-------|
+|  6 | 13 | Modifications in Version 5.5.1 | data may not have been supplied in the transmission. (see <<FAMILY_RECORD>>, page 24.) | data may not have been supplied in the transmission (see <<FAM_RECORD>>, page 24). | FAMILY_RECORD should be FAM_RECORD |
+| 32 | 36 | FAMILY_EVENT_STRUCTURE | n RESI | n RESI     {1:1} | Add cardinality |
+| 37 |  6 | MULTIMEDIA_LINK |            | [ | Add open bracket before first line of MULTIMEDIA_LINK |
+| 37 | 12 | MULTIMEDIA_LINK |            | ] | Add close bracket after last line of MULTIMEDIA_LINK |
+| 37 |  8 | MULTIMEDIA_LINK | n **OBJE** | n **OBJE**     {1:1} | Add cardinality |
+| 59 |  9 | PLACE_PHONETIC_VARIATION | the same form as  was the place name | the same form as was the place name | Remove extra space before "was" |
```

Comment:
```suggestion
| 59 |  9 | PLACE_PHONETIC_VARIATION | the same form as \ was the place name | the same form as was the place name | Remove extra space before "was" |
```

This change makes one of the spaces an non-breaking space, so that on an HTML page the double space is visible.
Other options would be to use an em space (U+2003) instead of a space (U+0020), or add a white-space:pre span around the two spaces.

Reply:
I don't follow.  Why should this be different from any other pair of words separated by a single space?

Reply:
This error is that there is a doubled space. But when rendered as HTML, I think that the original text "`as  was`" and the corrected text "`as was`" will appear to be identical because HTML defaults to `white-space: collapse` styling. I'm proposing that we find a way to make the doubled space visible on the webpage: a nbps, or bigger space character, or instruction to browsers to not collapse the white space.

Alternatively, since this error has no semantic impact at all, we could chose not to include it in the errata